addons/bus/static/src/js/services/bus_service.js

```diff
@@ -1,35 +1,25 @@
 odoo.define('bus.BusService', function (require) {
 "use strict";
 
-var bus = require('bus.bus').bus;
-
-var AbstractService = require('web.AbstractService');
+var CrossTab = require('bus.CrossTab');
 var core = require('web.core');
+var ServicesMixin = require('web.ServicesMixin');
+
+var BusService =  CrossTab.extend(ServicesMixin, {
+    dependencies : ['local_storage'],
+
+    // properties
+    _audio: null,
 
-var BusService =  AbstractService.extend({
-    name: 'bus_service',
     /**
-     * @override
+     * @abstract
```

Comment:
_"This method is necessary in order for this Class to be used to instantiate services"_